daemon/delete.go

```diff
@@ -79,7 +79,12 @@ func (daemon *Daemon) rmLink(container *container.Container, name string) error
 func (daemon *Daemon) cleanupContainer(container *container.Container, forceRemove, removeVolume bool) (err error) {
 	if container.IsRunning() {
 		if !forceRemove {
-			err := fmt.Errorf("You cannot remove a running container %s. Stop the container before attempting removal or use -f", container.ID)
+			state := container.StateString()
+			procedure := "Stop the container before attempting removal or use -f"
```

Comment:
Yeah, I was thinking if it was practically possible. Adding headers for this is probably taking it too far. We can update the message to include the state, and add the extra steps to the client, based on that 

Or perhaps just make the `-f` more generic ("or force remove" - possibly you have other suggestions for the wording)

@vdemeester do you have suggestions on this?